cpp/WKTJsiWorkletContext.h

```diff
@@ -92,15 +92,24 @@ class JsiWorkletContext
                   std::function<void(std::function<void()> &&)> jsCallInvoker);
 
   /**
-   Static / singleton default context
+   Get the default context
    */
-  static std::shared_ptr<JsiWorkletContext> getDefaultInstance() {
+  static std::shared_ptr<JsiWorkletContext> getDefaultInstanceAsShared() {
```

Comment:
```suggestion
  static const std::shared_ptr<JsiWorkletContext>& getDefaultInstanceAsShared() {
```

Reply:
shared_ptr's should never be passed as value - the whole point is to create a new shared pointer to keep track of reference counting etc. If it can be optimised (like just pass by reference without creating one for the result and one for the caller) - this is always done by the compiler if you return by value.

Reply:
Ah interesting, I didn't know that the compiler optimizes that.

Reply:
A lot of operations around std::move and smart pointer return optimisations are so easy for the compiler that it can warn you about not using the defaults :)